src/GitHubExtension/DataManager/GitHubDataManager.cs

```diff
@@ -703,6 +725,41 @@ private async Task UpdateIssuesAsync(Repository repository, Octokit.GitHubClient
         Issue.DeleteLastObservedBefore(DataStore, repository.Id, DateTime.UtcNow - LastObservedDeleteSpan);
     }
 
+    // Internal method to update releases. Assumes Repository has already been populated and created.
+    // DataStore transaction is assumed to be wrapped around this in the public method.
+    private async Task UpdateReleasesAsync(Repository repository, Octokit.GitHubClient? client = null, RequestOptions? options = null)
+    {
+        options ??= RequestOptions.RequestOptionsDefault();
+
+        // Limit the number of fetched releases.
+        options.ApiOptions.PageCount = 1;
+        options.ApiOptions.PageSize = 50;

```

Comment:
Generally we want to limit the scrolling of the widgets. For github we decided on 10.  Last 10 releases seems reasonable to me, so the page should be 10.